### PR TITLE
School library UI

### PR DESCRIPTION
**Pull request by sameedmohsin28**
In this pull request, I have created a basic CLI UI to interact with the school library app.

### CODE REVIEW

File: classes/app.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+end
+
+def create_student
+  puts 'Please enter the following details for the student'
+  print 'Name: '
+  student_name = gets.chomp
+  print 'Age: '
+  student_age = gets.chomp
+  print 'Classroom: '
+  student_classroom = gets.chomp
```
Comment (Zuhaib042) [inline classes/app.rb]: - [ ] Kindly consider removing this as asking input for classroom is not required in the example video.

File: classes/person.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  def initialize(age, name, parent_permission)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+    self.class.all_instances.push(self)
+  end
+
```
Comment (Zuhaib042) [inline classes/person.rb]: - [ ] There is a bug in your app. whenever I try to create a student or a teacher. it gives me an error that goes back to line 16 of this class.
I have checked that your books are getting created, but neither the teacher or student getting created, so I cant check the rentals as well. kindly consider fixing this bug. I am attaching a video for demonstration.

https://github.com/sameedmohsin28/OOP-school-library/assets/96683843/6331f1c5-2568-4ea0-b964-aef230384400

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @sameedmohsin28 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Linters passing
✅ Professional Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @sameedmohsin28 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**